### ISSUE TITLE
Install bug on win8

### ISSUE THREAD

**Issue by gaobb** [NikolaiT/GoogleScraper#141]
Hi, Can't get this to install on windows. I get the following: 

File "C:\Python34\lib\distutils\util.py", line 127, in convert_path raise ValueError, "path '%s' cannot end with '/'" % pathname 
ValueError: path './' cannot end with '/'

**Comment by ITSecMedia** [comment]
I experience the same issue on Windows 10 x64 with Anaconda virtual environment Python 3.5.1
Occurs on manual python setup.py install too

**Comment by derekcoleman** [comment]
Same issue. Any workaround/solution for this?

**Comment by ddmee** [comment]
Yeah, there is a work around. This is the same issue that is in issue [https://github.com/NikolaiT/GoogleScraper/issues/145 ] 

Download the source code from Github. Then you need to edit setup.py file. Remove the / from the package_dir={'examples': 'examples/'}. Once the forward slash is removed, save file. And then the installation should work on Windows.

You can install local source code using pip:

pip install <path>

**Comment by rifqiabidin** [comment]
Especially for supernoob ~(like me)~
As mentioned before, you can install local source code after download and modify the source code from Github.

Check this link if you still have no idea what to do
[https://packaging.python.org/installing/#installing-from-a-local-src-tree](https://packaging.python.org/installing/#installing-from-a-local-src-tree)

**Comment by eolica** [comment]
Thanks a lot!! I followed @rifqiabidin 's link and did the "Installing from local archives" and it finally worked.

**Comment by rifqiabidin** [comment]
@eolica glad to hear that
